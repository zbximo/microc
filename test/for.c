void main()
{
    Int i;
    for ( i=0; i<5; i++) {
        print i;
    }
}