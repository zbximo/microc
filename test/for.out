24 19 0 5 25 15 1 13 0 0 1 0 0 12 16 41 13 0 0 1 11 22 15 -1 15 0 13 0 0 1 9 11 10 9 11 0 1 1 12 15 -1 13 0 0 1 11 0 5 7 18 16 15 -1 21 -1